specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2022-08-11-preview/examples/Extension_List.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "location": "EastUS",
+    "publisher": "microsoft.azure.monitor",
+    "extensionType": "azuremonitorlinuxagent",
+    "version": "1.9",
+    "api-version": "2022-08-11-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "Extension": [
```

Comment:
The example contained this "Extension" inside of the return body but this is not defined in the definition. Can you correct one of those?

Reply:
fixed